lib/oli/groups/community.ex

```diff
@@ -6,15 +6,15 @@ defmodule Oli.Groups.Community do
     field :name, :string
     field :description, :string
     field :key_contact, :string
-    field :prohibit_global_access, :boolean
+    field :global_access, :boolean, default: true
```

Comment:
Add null: false to protect against nil please

Reply:
Added in the migration. What we can do here is call `validate_required` in the changeset function if that works for you

Reply:
No need it will always have a value.